modules/core/src/main/scala/com.snowplowanalytics.iglu/client/resolver/registries/JavaNetLookup.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2014-2023 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.iglu.client.resolver.registries
+
+import cats.effect.Sync
+import cats.Id
+import cats.data.OptionT
+import cats.implicits._
+import io.circe.Json
+import io.circe.parser.parse
+
+import com.snowplowanalytics.iglu.core.circe.implicits._
+import com.snowplowanalytics.iglu.core.{SchemaKey, SchemaList}
+
+import java.net.UnknownHostException
+import scala.util.control.NonFatal
+
+object JavaNetRegistryLookup {
```

Comment:
Could we make the filename same as this `object` name? So both either `JavaNetLookup` or `JavaNewRegistryLookup`